news/2024/2024-06-03-the-followpoint-eriha-top-drum-basher.md

```diff
@@ -0,0 +1,102 @@
+
+*What do you think of osu!taiko and its community in its current state?*
+
+> Recently there have been a lot of surprising events such as no mod pp record and for the first time the 2nd place in TWC is a non-Asian country, but on the other hand, there have been more and more discussions and problems about SR issues and pp.
+>
+> I believe that it's difficult for all players to get along, but we hope that many players will be able to interact and get along with players from all over the world through this game!
+
+*Any advice you would give to people who want to or just started playing osu!taiko?*
+
+> It's very difficult for someone just starting out to improve quickly, although there are individual differences.
+>
+> What I think is important is to step up. I'm all about stepping up and having goals. Even small goals, such as FCing a map you like or reaching your pp goal, are fine.
+>
+> But the most important thing is to enjoy this game to the fullest, so don't forget to smile and do your best! I support all players!
+
+*What is your favourite osu!taiko score you've set?*
+
+> My favorite score is [Period. ~ Seishin no Kousoku to Jiyuu o Tsukamu Jouka [Inner Oni]](https://osu.ppy.sh/scores/1871108211) where I have gotten PP record! This is the map I played the most and I think it is the record I am most proud of, as I was able to maintain a high acc while memorizing the placement of the notes!
+>
+> I also like the record of [rog-limitation [Insane]](https://osu.ppy.sh/scores/1875291020) best on the converted map. These burst maps are recorded with a key-scrubbing play style and are very habit-forming to play! If you are concerned about the hand, please refer to [this video](https://www.youtube.com/watch?v=UstoThJk02c)!
+
+*Who would you call the best osu!taiko player(s) at the moment and why?*
+
+> The best player I can think of right now is [SilverTyrant](https://osu.ppy.sh/users/14356353). He also competed in TWC2024 and I think it's very cool that he has recorded great scores on various maps, not just speed maps.
+>
+> Especially [Kage no Nai Hito](https://osu.ppy.sh/beatmapsets/1519045#taiko/3110395)'s [DT record](https://osu.ppy.sh/scores/1876120539) is so crazy that no player will surpass his record anytime soon.
+>
+> In addition, [sourufire](https://osu.ppy.sh/users/13249305) is probably the most growing of the Japanese players. His acc on the various speed maps is excellent and I look forward to competing with him in the future!
+
+*What do you feel is the most important aspect to keep motivation even when you are not improving, and what keeps you specifically motivated to play?*
+
+> I make it a point to take a break when my motivation wanes and observe the records of other players. Seeing other players happy to score well often motivates me naturally.
+>
+> I also make it a point to play other games and create cool down time. Lately I have been playing VALORANT with friends. I'm not good at fps, so I always lose lol.
```

Comment:
is it "cool down time" or "cooldown time"?
"cool down time" is down time that's cool (like playing valorant with friends is cool), and "cooldown time" would in this context mean taking breaks

Reply:
```suggestion
> I also make it a point to play other games and create cooldown time. Lately I have been playing VALORANT with friends. I'm not good at fps, so I always lose lol.
```